src/utils/TranslateMonths.ts

```diff
@@ -0,0 +1,17 @@
+export const translateMonth = (month: string) => {
+  const months: Record<string, string> = {
+    january: "Enero",
+    february: "Febrero",
+    march: "Marzo",
+    april: "Abril",
+    may: "Mayo",
+    june: "Junio",
+    july: "Julio",
+    august: "Agosto",
+    september: "Septiembre",
+    october: "Octubre",
+    november: "Noviembre",
+    december: "Diciembre",
+  };
+  return months[month.toLowerCase()] || month;
+};
```

Comment:
La definicion de la constante la haria por fuera del metodo, porque sino vas a estar inicializando la constante siempre que uses este metodo, por fuera la definicion se haria una unica vez


```suggestion
  const months: Record<string, string> = {
    january: "Enero",
    february: "Febrero",
    march: "Marzo",
    april: "Abril",
    may: "Mayo",
    june: "Junio",
    july: "Julio",
    august: "Agosto",
    september: "Septiembre",
    october: "Octubre",
    november: "Noviembre",
    december: "Diciembre",
  };
  
export const translateMonth = (month: string) => {
  return months[month.toLowerCase()] || month;
};
```